apps/opik-frontend/src/components/pages-shared/traces/FeedbackScoresEditor/FeedbackScoresEditor.tsx

```diff
@@ -33,10 +39,26 @@ type FeedbackScoresEditorFooterProps = {
   entityCopy: string;
 };
 
-const FeedbackScoresEditorHeader: React.FC = () => {
+type FeedbackScoresEditorHeaderProps = {
+  isTrace?: boolean;
+  isThread?: boolean;
+};
+
+const FeedbackScoresEditorHeader: React.FC<FeedbackScoresEditorHeaderProps> = ({
+  isTrace = false,
+  isThread = false,
+}) => {
+  let title = "Your scores";
+  if (isThread) {
+    title = "Your thread scores";
+  } else if (isTrace) {
+    title = "Your trace scores";
+  } else {
+    title = "Your span scores";
+  }
```

Comment:
it would be better to make a function and do something like this
```
const getTitleOfScores = ({ isThread, isTrace }) => {
  if (isThread) {
    return ...;
  }

  if (isTrace) {
    return ...;
  }

  return "Your span scores"
```

it's never gonna be "Your scores"